### PR TITLE
labwc: update to 0.7.0

### PR DESCRIPTION
**Pull request by tranzystorekk**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by tranzystorekk** [comment]
I've run into crashes when launching `wezterm`, `rio`, `gitk` at the moment.

**Comment by ahesford** [comment]
I cannot reproduce your problem. I'm running a branch four commits ahead of 0.7.0, none of which would have any bearing on this problem, and can launch all three programs.

File: srcpkgs/labwc/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'labwc'
 pkgname=labwc
-version=0.6.6
+version=0.7.0
 revision=1
 build_style=meson
 hostmakedepends="pkg-config scdoc wayland-devel"
-makedepends="cairo-devel glib-devel libpng-devel librsvg-devel libxml2-devel pango-devel wlroots0.16-devel"
+makedepends="cairo-devel glib-devel libpng-devel librsvg-devel libxml2-devel pango-devel wlroots0.17-devel"
```
Comment (ahesford) [inline srcpkgs/labwc/template:7]: Wrap this at 78 characters.

File: srcpkgs/labwc/template [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 short_desc="Wayland window-stacking compositor"
 maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://labwc.github.io/"
 changelog="https://raw.githubusercontent.com/labwc/labwc/master/NEWS.md"
 distfiles="https://github.com/labwc/labwc/archive/refs/tags/${version}.tar.gz"
-checksum=fea73450a0cc806339e38414973261812f6376d09cf387e693a4bddf75c40916
+checksum=2607219cb766db4d258a20c62c98ea16cd6d62c735e65f35d916188cf0ccbe70
 
 post_install() {
```
Comment (ahesford) [inline srcpkgs/labwc/template:16]: All of the files installed by `post_install` are already installed in `/usr/share/doc/labwc`. I think this function should just be dropped.

**Review by ahesford — CHANGES_REQUESTED** [review]

**Comment by tranzystorekk** [comment]
I'm wondering if we should disregard the vm-specific problems and just merge this anyway...

**Comment by ahesford** [comment]
I agree that problems in VMware shouldn't block the update, especially because it seems to originate outside of labwc.

**Pull request merged**